src/util/contentful/richTextRenderer.js

```diff
@@ -142,9 +144,6 @@ export function richTextRenderer(content) {
 			[BLOCKS.EMBEDDED_ASSET]: node => {
 				return assetRenderer(node?.data?.target?.fields);
 			},
-			[BLOCKS.DOCUMENT]: (node, next) => {
```

Comment:
This is unnecessary, we just need a single wrapping div of <div class="tw-prose tw-whitespace-pre-wrap">, which is in: src/components/Contentful/DynamicRichText.vue